content/topics/How-to-ask-for-help-with-your-code/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: how to ask for help with your code
+ready: true
+---
+
+A big part of learning to be a programmer of any kind is knowing how to get help. This means knowing how to ask questions.
+Your questions should be short and clear. Your question should include all the information that is needed to answer the question.
+Practice asking good questions. It's like a superpower. If you are easy to help then people will help you.<br/><br/>
+
+### These also include:
+
+#### how to ask good questions on slack:
+
+* all Slack questions should be addressed on the slack **Tech Assist** channel.
+ - This helps you get to a broad audience from the tech department (.e.g Recruits, Managers and Alumni).
+ - Please don't send screenshots what you can do is use backtick (` `) characters to represent a code form.
+ - Write full error messages. (Summarise if you can and make sure you understand the error before sending it)
+ - If your issues are not addressed on the "Tech Assist" Channel, then run your issue with your specific department channel, then if it doesn't help then run to your **cohort mentors**.
+ **NB. IT IS YOUR RESPONSIBILITY TO HELP OTHERS WHEN AND IF YOU HAVE THE ANSWERS** (Basically help and be helped)  
+
+#### Things to note
```

Comment:
This content is good! 

I would refactor it slightly to make it a bit clearer.

Maybe a section on
"Characteristics of a good question" or something
- all the info and context
- evidence of effort
- no bloody screenshots
- actual error messages instead of  "I got an error"

And then a section on "Where to ask good questions". 
- slack + backticks
- stack-overflow == friggin awesome